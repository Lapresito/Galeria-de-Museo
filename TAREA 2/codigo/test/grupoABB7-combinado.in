crearGrupoABB
liberarGrupoABB
crearGrupoABB
crearVisitante 10 Brunilda Gomez 34
agregarAGrupoABB
crearVisitante 20 Casimira Ruiz 22
agregarAGrupoABB
crearVisitante 16 Zenobia Garcia 40
agregarAGrupoABB
crearVisitante 24 Brunilda Perez 16
agregarAGrupoABB
crearVisitante 30 Zenobia Gonzalez 22
agregarAGrupoABB
crearVisitante 31 Rufino Fernandez 24
agregarAGrupoABB
crearVisitante 32 Casimira Perez 16
agregarAGrupoABB
crearVisitante 40 Ubaldo Garcia 21
agregarAGrupoABB
crearVisitante 8 Esmeralda Fernandez 23
agregarAGrupoABB
crearVisitante 4 Filemon Lopez 42
agregarAGrupoABB
crearVisitante 6 Casimira Martinez 18
agregarAGrupoABB
crearVisitante 7 Rufino Martinez 25
agregarAGrupoABB
alturaGrupoABB
existeEnGrupoABB 6
existeEnGrupoABB 31
existeEnGrupoABB 2
existeEnGrupoABB 18
maxIdGrupoABB
edadPromedioGrupoABB
removerDeGrupoABB 30
removerDeGrupoABB 8
removerDeGrupoABB 31
removerDeGrupoABB 24
alturaGrupoABB
existeEnGrupoABB 6
existeEnGrupoABB 31
maxIdGrupoABB
edadPromedioGrupoABB
liberarGrupoABB
Fin
