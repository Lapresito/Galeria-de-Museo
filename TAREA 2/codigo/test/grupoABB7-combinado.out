Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> Grupo liberado con exito.
3> El grupoABB ha sido creado de forma exitosa.
4> Visitante creadox de forma exitosa.
5> Se ha agregado el visitante al grupo de forma exitosa.
6> Visitante creadox de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Visitante creadox de forma exitosa.
9> Se ha agregado el visitante al grupo de forma exitosa.
10> Visitante creadox de forma exitosa.
11> Se ha agregado el visitante al grupo de forma exitosa.
12> Visitante creadox de forma exitosa.
13> Se ha agregado el visitante al grupo de forma exitosa.
14> Visitante creadox de forma exitosa.
15> Se ha agregado el visitante al grupo de forma exitosa.
16> Visitante creadox de forma exitosa.
17> Se ha agregado el visitante al grupo de forma exitosa.
18> Visitante creadox de forma exitosa.
19> Se ha agregado el visitante al grupo de forma exitosa.
20> Visitante creadox de forma exitosa.
21> Se ha agregado el visitante al grupo de forma exitosa.
22> Visitante creadox de forma exitosa.
23> Se ha agregado el visitante al grupo de forma exitosa.
24> Visitante creadox de forma exitosa.
25> Se ha agregado el visitante al grupo de forma exitosa.
26> Visitante creadox de forma exitosa.
27> Se ha agregado el visitante al grupo de forma exitosa.
28> La altura del grupo es 7.
29> El visitante con id 6 pertenece al grupo.
30> El visitante con id 31 pertenece al grupo.
31> El visitante con id 2 NO pertenece al grupo.
32> El visitante con id 18 NO pertenece al grupo.
33> El mayor id en el grupo es 40.
34> La edad promedio del grupo es 25.25.
35> El visitante con id 30 se removio del grupo.
36> El visitante con id 8 se removio del grupo.
37> El visitante con id 31 se removio del grupo.
38> El visitante con id 24 se removio del grupo.
39> La altura del grupo es 4.
40> El visitante con id 6 pertenece al grupo.
41> El visitante con id 31 NO pertenece al grupo.
42> El mayor id en el grupo es 40.
43> La edad promedio del grupo es 27.25.
44> Grupo liberado con exito.
45> Fin.
